src/index.ts

```diff
@@ -18,6 +18,15 @@ type Dependencies = Record<string, unknown>;
 // TODO: add support for `openapi-ts.config.ts`
 const configFiles = ['openapi-ts.config.js', 'openapi-ts.config.cjs', 'openapi-ts.config.mjs'];
 
+// Mapping of all dependencies required by each client. These should be installed in the generated client package
+const clientDependencies: Record<Config['client'], string[]> = {
+    angular: ['@angular/common', '@angular/core', 'rxjs'],
```

Comment:
I thought CLI is used for Angular projects? https://angular.io/guide/setup-local

Reply:
Possibly, but the generated client for angular only imports code from these packages (so cli is not required). These are a map of dependencies that the client we generate needs.